src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -699,20 +727,26 @@ private static void PrintQueryAttempts(List<Worker> workers, WorkloadConfigurati
         workerTotalTasks[sample.tranType - 1]++;
       }
     }
+    
+    DecimalFormat df = new DecimalFormat();
+    df.setMaximumFractionDigits(2);
+
     for (int i = 0; i < numTxnTypes; ++i) {
       String op = transactionTypes.get(i + 1);
       int totalTasks = workerTotalTasks[i];
       resultOut.append(String.format("%14s |%10d |", op, totalTasks));
+      List<String> valueList = new ArrayList<> ();
       for (int retry = 0; retry < numTriesPerProc; ++retry) {
         int numRetries = totalFailedTries[i][retry];
         double pctRetried = numRetries;
         pctRetried /= totalTasks;
         pctRetried *= 100.0;
         resultOut.append(String.format("%16d (%5.2f%%) |", numRetries, pctRetried));
+        valueList.add(numRetries + " (" + df.format(pctRetried) + "%)");
```

Comment:
seems a bit clunky that we add strings to a value list just to later pass the value list to jsonMetricsHelper.addRetry

could we instead have a method that accepts `jsonMetricsHelper.addRetryData(op, totalTasks, totalFailedTries[i])` and is called once at L747?

Reply:
The reason storing this in a string list to get the percentage failure as well, this is not part of the totalFailedTries array.

Reply:
We will display the failure count.